lib/gtl_formatter.rb

```diff
@@ -27,7 +27,8 @@ class GtlFormatter
   ].freeze
 
   COLUMN_WITH_IMAGE = {
-    'ext_management_system.name' => 'fonticon_or_fileicon'
+    'ext_management_system.name' => 'fonticon_or_fileicon',
+    'resource.name'              => 'fonticon_or_fileicon',
```

Comment:
My comment was lost, but I am concerned about this because of how generic t is - let's see if we can get actual ext_management_system relationships in core, and then this should Just Work